docs/setup/k8s/running-in-production-k8s.md

```diff
@@ -188,7 +188,7 @@ EOF
 > Otherwise, Kaniko will fail to push the image to ECR because the image name is being determined during the build process.
 
 Create the ECR token secret to be used as `imagePullSecret` of function pods.
-Since ECR tokens go stale after 12 hours, the secret must be refreshed periodically (can be done with a cron job as described in [this blogpost](https://skryvets.com/blog/2021/03/15/kubernetes-pull-image-from-private-ecr-registry/#update---aws-ecr-token-refresh))
+Since ECR tokens go stale after 12 hours, the secret must be refreshed periodically (can be done with a cron job as described in [this page](https://skryvets.com/blog/2021/03/15/kubernetes-pull-image-from-private-ecr-registry/#update---aws-ecr-token-refresh))
```

Comment:
It is a blog post, just need a space
```suggestion
Since ECR tokens go stale after 12 hours, the secret must be refreshed periodically (can be done with a cron job as described in [this blog post](https://skryvets.com/blog/2021/03/15/kubernetes-pull-image-from-private-ecr-registry/#update---aws-ecr-token-refresh))
```